python/amici/ode_export.py

```diff
@@ -2209,9 +2425,12 @@ def __init__(
         self.model_path: str = os.path.abspath(output_dir)
         self.model_swig_path: str = os.path.join(self.model_path, 'swig')
 
+        # We need to process events and Heaviside functions in the ODE Model,
+        # before adding it to ODEExporter
+        ode_model.get_events()
```

Comment:
calling `get_events` without using the return value feels wrong. rename?

Reply:
Feels like you're right. Renamed to parse_events()